views/home-sidebar.pug

```diff
@@ -2,10 +2,10 @@ h4.sidebar-item#sidebar-name #{user.fname} #{user.lname}
 h3.sidebar-item#sidebar-points #{user.points} Points
 p.sidebar-item Questions attempted: #{user.attemptedCount}
 p.sidebar-item Questions successfully answered: #{user.answeredCount}
-if user.attempted == 0
+if (user.answeredCount + user.attemptedCount) === 0
```

Comment:
Don't think this change is necessary, isn't the answeredCount included in the attemptedCount?

Reply:
attempted  = number of students who attempted to answer with wrong answer
answered =  number of students who answered correctly

Reply:
Ok I see, thanks for clarifying this.

Reply:
yeah, I agree that the naming is a bit confusing. Lets take this offline and we can change this. nice catch

Reply:
There should definitely be comments explaining this.

Reply:
Good point. I think it will be better to document this once I get the individual points per question working. If I add comments now, I will have to change them later.